AGENTS.md

```
@@ -252,7 +252,40 @@ Business logic is extracted into service objects and models:
 - Tests in spec/requests/ generate documentation
 - Run `just docs-api` or `RAILS_ENV=test rails rswag` to regenerate
 
-### Testing Strategy
+#### JavaScript Testing
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)